src/common/types/chainner-scope.ts

```diff
@@ -51,7 +51,9 @@ struct PyTorchModel {
     arch: string,
     size: string,
     subType: string,
+    tiling: ModelTiling,
 }
+enum ModelTiling { Supported, Discouraged, Internal }
```

Comment:
Why does this have to exist when other enums don't require it?

Reply:
What do you mean? Are you asking why I used an explicit enum here instead of a union of string literals? If so, because I wanted to closely match spandrel's `ModelTiling` enum. I could have used string literals, but I thought that an enum was the cleaner solution.